### ISSUE TITLE
trailing_comma rule false positive for comments

### ISSUE THREAD

**Issue by roddi** [realm/SwiftLint#940]
Hi,

the following code will create a false positive in 0.13.1:

```
let example = [ "a",
                "b"
                // "c",
        ]
```
